lib/iris/tests/unit/util/test_reverse.py

```diff
@@ -94,76 +99,108 @@ def setUp(self):
             data, dim_coords_and_dims=[(a2, 0), (b2, 1)]
         )
 
-    def test_cube_dim(self):
+    def test_cube_dim0(self):
         cube1_reverse0 = reverse(self.cube1, 0)
-        cube1_reverse1 = reverse(self.cube1, 1)
-        cube1_reverse_both = reverse(self.cube1, (0, 1))
 
         self.assertArrayEqual(self.cube1.data[::-1], cube1_reverse0.data)
         self.assertArrayEqual(
             self.cube2.coord("a").points, cube1_reverse0.coord("a").points
         )
+        self.assertArrayEqual(
+            self.cube2.coord("a").bounds, cube1_reverse0.coord("a").bounds
+        )
         self.assertArrayEqual(
             self.cube1.coord("b").points, cube1_reverse0.coord("b").points
         )
 
+    def test_cube_dim1(self):
+        cube1_reverse1 = reverse(self.cube1, 1)
+
         self.assertArrayEqual(self.cube1.data[:, ::-1], cube1_reverse1.data)
         self.assertArrayEqual(
             self.cube1.coord("a").points, cube1_reverse1.coord("a").points
         )
+        self.assertArrayEqual(
+            self.cube1.coord("a").bounds, cube1_reverse1.coord("a").bounds
+        )
         self.assertArrayEqual(
             self.cube2.coord("b").points, cube1_reverse1.coord("b").points
         )
 
+    def test_cube_dim_both(self):
+        cube1_reverse_both = reverse(self.cube1, (0, 1))
+
         self.assertArrayEqual(
             self.cube1.data[::-1, ::-1], cube1_reverse_both.data
         )
         self.assertArrayEqual(
             self.cube2.coord("a").points, cube1_reverse_both.coord("a").points
         )
+        self.assertArrayEqual(
+            self.cube2.coord("a").bounds, cube1_reverse_both.coord("a").bounds
+        )
         self.assertArrayEqual(
             self.cube2.coord("b").points, cube1_reverse_both.coord("b").points
         )
 
-    def test_cube_coord(self):
+    def test_cube_coord0(self):
```

Comment:
I'm not super keen that we've repeated some code from `test_cube_dim0` identically here (c.f. to rest of file) - particularly as changing one instance would require that the other changed. The assertions could jump out to an auxiliary method that's called by both tests?

Reply:
how's this? https://github.com/SciTools/iris/pull/4466/commits/0dea5af6078d04209e35630e9dde7d55910eaae1